index.html

```diff
@@ -2,11 +2,146 @@
+        <h3>Магазин Глэйси – это онлайн и офлайн-магазин по продаже мороженого собственного производства на развес</h3>
+          <ul>
+              <li>
+                   Все наше мороженое изготавливается на собственном производстве, с использованием современного оборудования и проверенных временем технологий.
+              </li>
+              <li>
+                   Для приготовления мороженого используются настоящие сливки и молоко высочайшего качества. Все дополнительные ингредиенты и добавки произведены из натурального,<br>
+                   экологически чистого сырья.
+             </li>
+             <li>
+                  Закупка ингредиентов производится только у проверенных фермерских хозяйств и компаний, с которыми нас связывает долговременное сотрудничество.
+             </li>
+             <li>
+                  Доставка нашего мороженого до заказчиков осуществляется в специальном термопаке, который не дает мороженому растаять в пути и позволяет сохранить превосходный вкус.
+             </li>
+           </ul>
+       </section>
+     <section>
+         <!--Новости -->
+         <p>Новое в нашем блоге</p>
+         <a href="blank.html">10 способов сервировки фруктовых щербетов к столу</a>
+       </section>
+     <section>
+         <!--отправка подписки -->
+         <p>Подпишитесь на нашу сладкую рассылку и будьте всегда в курсе всего самого вкусного, что у нас происходит. Обещаем не спамить и не слать всякой ненужной ерунды. Честно =)</p>
+         <input type="email" placeholder="email@example.com">
+         <input type="submit" value="ОТПРАВИТЬ">
+     </section>
+     <section>
+        <!--Контактная информация-->
+        <p>
+          Адрес главного офиса<br>
+          и офлайн-магазина:
+        </p>
+        <p>
+          наб. Реки Карповки,<br>
+          5 лит П, Санкт-Петербург
+        </p>
+        <p>Для заказов по телефону:</p>
+        <p><a href="tel:+78005558628">+7 800 555-86-28</a></p>
+        <p>(с 10 до 20 ежедневно)</p>
+        <button type="button">Форма обратной связи</button>
+      </section>
+    </main>
+   <footer>
+     <section>
+       <ul>
+         <li>
+           <a href="#">Фейсбук</a>
+         </li>
+         <li>
+           <a href="#">ВКонтакте</a>
+         </li>
+         <li>
+           <a href="#">Инстаграм</a>
+         </li>
+         <li>
+           <a href="#">Твитер</a>
+         </li>
+       </ul>
+     </section>
+     <section>
+       <ul>
+         <li>
+           <a href="blank.html">Для поставщиков</a>
+         </li>
+         <li>
+           <a href="blank.html">О производстве</a>
+         </li>
+         <li>
+           <a href="blank.html">Наши документы</a>
+         </li>
+         <li>
+           <a href="blank.html">Экологические стандарты</a>
+         </li>
+       </ul>
+     </section>
+     <section>
+       <p><a href="https://htmlacademy.ru/"><img src="" width="107.89" height="30.99" alt=""></a></p>
```

Comment:
Значением атрибутов `width` и `heigth` должны быть только целые числа. Дробные нужно округлять:
```suggestion
       <p><a href="https://htmlacademy.ru/"><img src="" width="108" height="31" alt=""></a></p>
```